### DISCUSSION TITLE
How do I restore the default interfaces??

### DISCUSSION THREAD

**Discussion by fr0zensphere** [smplayer-dev/smplayer#97] [Q&A] [answered]
Messed around with some options and somehow all the toolbars went away, and they're not coming back. 

How do I bring it back to the default state, like this? 

![smplayer GUI](https://user-images.githubusercontent.com/23515872/118407224-c5ab3180-b64d-11eb-8059-116fd2b8a804.png)

This is extremely frustrating. Unbelievable that I can't immediately find the option to restore it... this should be easy. And I can't even find it by searching the web.

**Comment by smplayer-dev** [comment, marked as answer]
Maybe you enabled compact mode. Press Ctrl+C to turn it off.

**Reply by fr0zensphere** [reply]
Ah yes, that did it, thank you!

**Reply by fr0zensphere** [reply]
I think it would benefit the software if the toggle to turn this on/off were available in the right-click menu under View or in the Preferences window.
